environment.py

```diff
@@ -733,5 +733,12 @@ def env() -> Mapping[str, Optional[str]]:
         #     'channel_id': 'your-channel-id'
         # })
         #
-        'azul_slack_integration': None
+        'azul_slack_integration': None,
+
+        # The maximum number of requests blocked by the WAF during an evaluation
+        # period before an alarm is raised. The length of the evaluation period
+        # can be found by looking at the WAF specific aws_cloudwatch_metric_alarm
+        # resource in the Terraform config files.
+        #
+        'AZUL_MAX_WAF_BLOCKED_REQUESTS': '0',
```

Comment:
Tying the threshold to a deployment is coincidental. Which deployment becomes victim to an attack is entirely out of our hands. It could hit production or a personal deployment. Ideally we'd make the metric a ratio between the numbers of served and blocked requests. If that's not possible we need to use a threshold that's "normal" for `prod`. In that case, please request a PL slot to discuss what that "norm" is.